test/DemoTests.m

```diff
@@ -41,6 +42,17 @@ function setupVivado(~)
     end
     
     methods(Test)
+        function buildHDLLoopbackDelayEstimation(testCase)
+            cd(fullfile(testCase.root,'trx_examples/targeting/loopback-delay-estimation'));
+            hdlworkflow;
+            if ~isempty(out)
```

Comment:
This is checking the variable `out` which is expected from the hdlworkflow script. You need to add something like: https://github.com/analogdevicesinc/TransceiverToolbox/blob/master/trx_examples/targeting/frequency-hopping/hdlworkflow.m#L171 to your hdlworkflow script